src/scancode/api.py

```diff
@@ -312,7 +312,7 @@ def get_file_info(location, **kwargs):
     return result
 
 
-def extract_archives(location, recurse=True, replace_originals=False):
+def extract_archives(location, recurse=True, replace_originals=False, ignored_extensions = ()):
```

Comment:
Nit picking again, I can spot a few extra spaces in `ignored_extensions = ()` ... may be rather `ignored_extensions=()`